vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -294,17 +315,19 @@ and
     [<ProvideEditorExtension(FSharpConstants.editorFactoryGuidString, ".fsscript", 97)>]
     [<ProvideEditorExtension(FSharpConstants.editorFactoryGuidString, ".ml", 97)>]
     [<ProvideEditorExtension(FSharpConstants.editorFactoryGuidString, ".mli", 97)>]
-    internal FSharpLanguageService(package : FSharpPackage) =
+    internal FSharpLanguageService(package : FSharpPackage)  as this =
```

Comment:
Using ``as this`` is a code smell, and I believe it's only needed because of the contorted use of an event above?